example/src/test-data-provider.ts

```diff
@@ -191,6 +214,25 @@ export class TestDataProvider {
                         nextPossibleState = state.startTime + state.duration - entry.startTime;
                     }
                 });
+
+                const _annotations = row.annotations;
+                if (!!_annotations) {
+                    _annotations.forEach((annotation: any, annotationIndex: number) => {
+                        const start = annotation.range.start - entry.startTime;
+                        if (range.start < start && range.end > start) {
+                            annotations.push({
+                                id: 'mark_' + rowIndex + '_' + annotationIndex,
+                                range: {
+                                    start: annotation.range.start - this.absoluteStart,
+                                    end: annotation.range.end - this.absoluteStart
+                                },
+                                label: '',
+                                data: {type: 'cross, I suppose?',}
```

Comment:
This is supposed to be {style: {parentKey: 'styleKey', values: {}}}
where 'styleKey' is defined in the style map returned by the /style endpoint, which contains symbol-type, color and height.

Reply:
I did something deliberately bad to have a test of the default method... what do you suggest?

Reply:
The style is what identifies it as a symbol. If style or symbol-type property is not present, it's supposed to mean 'none' e.g. not a symbol annotation. If color property is absent it should be #000000. If height property is absent it should be 1.0f e.g. symbol size (radius) = half of row height.